.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Issue must have
+
+<!--
+Customized from the [template](https://github.com/thealphadollar/vanilla-i18n/blob/master/.github/ISSUE_TEMPLATE.md)
```

Comment:
This also needs to be updated I believe, @denisvieira05

Reply:
done @thealphadollar